### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by illusional**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,67 +1,94 @@
...
+The conda env defines the development dependencies, and the pure API deps
+are defined in `requirements.txt`.
 
 ```shell
-# use localhost:8000
-export SM_ENVIRONMENT=LOCAL
- # use a local account to generate identity-token for auth
-export GOOGLE_APPLICATION_CREDENTIALS=/path/to/sa.json
+conda env create -f environment-dev.yml
+conda activate sample-metadata
```
Comment (vivbak) [inline README.md:53]: To match the name specified in `environment-dev.yml`
```suggestion
conda activate sample-metadata-dev
```

File: README.md [hunk 2]
```diff
@@ -74,59 +101,75 @@ Run a mariadb container that will server your database. `-p 3307:3306` remaps th
...
-### Running a file directly
+In a different terminal, execute the following
+request to create a new project called 'dev'
 
-Due to the way python imports work, you're unable to run files directly. To run files, you must run them as a module, for example, to run the `db/python/layers/sample.py` file directly (in case you put an `if __name__ == "__main__"` block in there), you could use the following:
+```shell
+curl -X 'PUT' \
+  'http://localhost:8000/api/v1/project/?name=dev&dataset=dev&gcp_id=dev&create_test_project=false' \
+  -H 'accept: application/json' \
+  -H "Authorization: Bearer $(gcloud auth-print-identity-token)"
```
Comment (vivbak) [inline README.md:160]:
```suggestion
  -H "Authorization: Bearer $(gcloud auth print-identity-token)"
```

**Review by vivbak — CHANGES_REQUESTED** [review]
Hey @illusional this is great, thanks a bunch! Just a couple typos :)

File: README.md (hunk 1 above)
Comment (illusional) [reply README.md:53]: This should've been changed to `sample-metadata` in environment-dev: https://github.com/populationgenomics/sample-metadata/blob/6bef596a9870f1caab39f8d11d7f2c80907720ff/environment-dev.yml

Comment (vivbak) [reply README.md:53]: oh, sorry I was in the wrong branch!

**Review by vivbak — APPROVED** [review]

**Pull request merged**